tags-input.js

```diff
@@ -190,25 +216,30 @@
 				select(selectedTag.nextSibling);
 			}
 			else {
+				// If there are no tags yet, update the original input as the user types -
+				// so that users who only want one thing don't have to enter commas
+				if ( ! $('.tag', true).length ) {
+					originalInput.value = getValue();
+					originalInput.dispatchEvent(new Event('originalInput'));
```

Comment:
Hmm, this won't work. `input` is an actual event in browsers, `originalInput` is not.